spec/spec_helper.rb

```diff
@@ -25,6 +26,8 @@
 
   config.example_status_persistence_file_path = "tmp/rspec_examples.txt"
   config.order = :random
+
+  config.include Helpers
```

Comment:
Not sure you need this. Or at least you can just define that for feature specs...

Reply:
Done in sha: 5e2fd92